frontend/src/hooks/useIsMobile/useIsMobile.js

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+
+const useIsMobile = (width) => {
```

Comment:
I asked to write a resize hook, it's more universal, but let it be, if necessary, we will rework it
Change the name, it's not just to check if it's a phone